### PR TITLE
ROLIE category and service document location

### PR DESCRIPTION
**Pull request by tschmidtb51**
- addresses parts of https://github.com/oasis-tcs/csaf/issues/632
- provide guidance where to expect a ROLIE category document
- provide guidance where to expect a ROLIE service document and state suggested filename

### CODE REVIEW

File: csaf_2.1/prose/edit/src/distributing.md [hunk 1]
```diff
@@ -288,6 +288,8 @@ having the `rel` value of `signature`.
 
 The use and therefore the existence of ROLIE service document is optional.
 If it is used, each ROLIE service document MUST be a JSON file that conforms with [cite](#RFC8322) and lists the ROLIE feed documents.
+It MAY also list the corresponding ROLIE category documents.
```
Comment (sthagen) [inline csaf_2.1/prose/edit/src/distributing.md:291]: I think that we often use MAY slightly different from what it is expected to signal.
My understanding of what we want to say here is that the listing of the corresponding ROLIE category documents is optional. Could we state this without a MAY?

**Review by sthagen — APPROVED** [review]
LGTM (only one question).

File: csaf_2.1/prose/edit/src/distributing.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.1/prose/edit/src/distributing.md]: Addressed in [9de0c75](https://github.com/oasis-tcs/csaf/pull/745/commits/9de0c7581a939d119adf9d9305520be921302bfd)

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by santosomar** [comment]
The motion to accept the pull request as suggested in https://github.com/oasis-tcs/csaf/pull/745 and include it into CSAF 2.1, has passed. It automatically carried on 2024-06-05 23:00 UTC.
https://groups.oasis-open.org/discussion/motion-for-745

**Review by santosomar — APPROVED** [review]

**Pull request merged**